rbac/management/group/view.py

```diff
@@ -573,6 +584,22 @@ def roles(self, request, uuid=None):
 
         return Response(status=status.HTTP_200_OK, data=response_data.data)
 
+    def ordered_principals(self, principals, order_field):
+        """Return principals ordered by order_field."""
+        all_fields = VALID_PRINCIPAL_ORDER_FIELDS + ['-' + field for field in VALID_PRINCIPAL_ORDER_FIELDS]
+        if order_field.lower() in all_fields:
```

Comment:
Overall LGTM! Maybe we could remove this and the VALID_PRINCIPAL_ORDER_FIELDS? Because 1. the 'sorted' would report KeyError if it is not a valid key to be sorted; 2. that would support sorting by all available fields including the 'is_active' added by Keith. Or we should have control over the fields to be sorted? Any thoughts?

Reply:
Hmm, these are some good points. I'll poke around a bit and see what I can do with it